### PR TITLE
CMake documentation builds

### PR DESCRIPTION
**Pull request by saadrahim**
This PR enables 
```
mkdir build
cd build
cmake ../
make doc
```

### CODE REVIEW

**Comment by saadrahim** [comment]
@MathiasMagnus please update the instructions?

File: README.md [hunk 1]
```diff
@@ -254,6 +261,26 @@ cd _build/html
...
+</summary>
+
+```shell
+# Configure the project
+cmake -S ./rocPRIM -B ./rocPRIM/build -D BUILD_DOCS=ON
+
+# Build the documentation
+cmake --build ./rocPRIM/build --target doc
+
+# For e.g. serve the HTML docs locally
```
Comment (saadrahim) [inline README.md:277]: Example to server the HTML docs locally

**Review by saadrahim — COMMENTED** [review]
minor phrasing change requested

**Review by stanleytsang-amd — APPROVED** [review]

**Comment by stanleytsang-amd** [comment]
@samjwu @saadrahim is this PR stale or still relevant?

**Comment by samjwu** [comment]
Updated this and created https://github.com/ROCm/rocPRIM/pull/583 because I am unable to push to the fork this PR is based on

**Comment by samjwu** [comment]
Merged in https://github.com/ROCm/rocPRIM/pull/583

**Pull request closed without merging**